### PR TITLE
Add EKS 1.30 note; Kubescaler fixes

### PR DESCRIPTION
**Pull request by chipzoller**
Signed-off-by: chipzoller <chipzoller@gmail.com>

## Related Issue

Related issues are listed in associated target PR #1067 

## Proposed Changes

* Adds a note for EKS 1.30+ users.
* Adds a fourth level heading under Prerequisites for installing CSI driver
* Makes Kubescaler page fixes

### CODE REVIEW

File: install-and-configure/advanced-configuration/controller/kubescaler.md [hunk 1]
```diff
@@ -33,15 +33,9 @@ Notable Helm values:
 
 ### Supported workload types
 
-Kubescaler supports:
+Kubescaler supports apps/v1 Deployments.
 
-* apps/v1 Deployments
-* apps/v1 DaemonSets
-* batch/v1 CronJobs (K8s v1.21+). No attempt will be made to autoscale a CronJob until it has run at least once.
```
Comment (thomasvn) [inline install-and-configure/advanced-configuration/controller/kubescaler.md:40]: Was there a recent discussion that sparked this change?

**Review by thomasvn — APPROVED** [review]

File: install-and-configure/advanced-configuration/controller/kubescaler.md (hunk 1 above)
Comment (chipzoller) [reply install-and-configure/advanced-configuration/controller/kubescaler.md:40]: Not really, I just noticed that the docs on this page were inconsistent. It says at the top that only deployments were supported, and in this section it contradicts that statement. Any problems you see with this?

Comment (thomasvn) [reply install-and-configure/advanced-configuration/controller/kubescaler.md:40]: Nope! After further research it looks like this change is correct according to our internal docs on it https://github.com/kubecost/cluster-controller/blob/develop/docs/kubescaler.md

Comment (chipzoller) [reply install-and-configure/advanced-configuration/controller/kubescaler.md:40]: Thanks for the checkup.

**Pull request merged**